bump-and-tag.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+gpgloader
+
+gpg --list-keys
+
+echo "Running bump, tag, and bump..."
+
+# Bump
+perl -pi -e "s/version = \"$CURRENT_VERSION\"/version = \"$NEXT_VERSION\"/" pyproject.toml
```

Comment:
Never thought I'd see `perl` in a PR 😅

Reply:
The version of `sed` in the container was problematic, I gave up and went with this.  :smile: